apps/mark-scan/accessible-controller/src/controllerd.rs

```diff
@@ -303,42 +336,109 @@ fn create_virtual_device() -> Device {
 }
 
 fn main() {
-    // Open the serial port
-    let port = serialport::new(DEVICE_PATH, DEVICE_BAUD_RATE)
-        .timeout(POLL_INTERVAL)
-        .open();
-
-    println!("Opened controller serial port at {DEVICE_PATH}");
+    log(Log {
+        event_id: "process-started".to_string(),
+        message: format!("Opened controller serial port at {DEVICE_PATH}"),
+        event_type: EventType::SystemAction,
+        ..Default::default()
+    });
+
+    let running = Arc::new(AtomicBool::new(true));
```

Comment:
I don't think we need this thread-safe bool but using it is consistent with `ctrlc` docs, so might as well